src/aap_eda/settings/default.py

```diff
@@ -654,29 +653,17 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
     "SECRET_KEY": settings.get("RESOURCE_SERVER__SECRET_KEY", ""),
     "VALIDATE_HTTPS": settings.get("RESOURCE_SERVER__VALIDATE_HTTPS", False),
 }
-RESOURCE_JWT_USER_ID = settings.get("RESOURCE_JWT_USER_ID", "")
-RESOURCE_SERVICE_PATH = settings.get(
-    "RESOURCE_SERVICE_PATH", "/api/gateway/v1/service-index/"
-)
-
-if DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED:
-    if (
-        RESOURCE_SERVER["URL"]
-        and RESOURCE_SERVER["SECRET_KEY"]
-        and RESOURCE_JWT_USER_ID
-    ):
-        RQ_PERIODIC_JOBS.append(
-            {
-                "func": "aap_eda.tasks.shared_resources.resync_shared_resources",  # noqa E501
-                "interval": 900,
-                "id": "resync_shared_resources",
-            }
-        )
-    else:
-        raise ImproperlyConfigured(
-            "RESOURCE_SERVER__URL, RESOURCE_SERVER__SECRET_KEY, "
-            "and RESOURCE_JWT_USER_ID settings must be properly configured"
-        )
+RESOURCE_JWT_USER_ID = settings.get("RESOURCE_JWT_USER_ID", None)
+RESOURCE_SERVICE_PATH = settings.get("RESOURCE_SERVICE_PATH", None)
+
+if RESOURCE_SERVER["URL"] and RESOURCE_SERVER["SECRET_KEY"]:
```

Comment:
Can `RESOURCE_SERVER` be an empty dict? if so it will raise `KeyError`

Maybe check using `.get` method?
```suggestion
if RESOURCE_SERVER.get("URL") and RESOURCE_SERVER.get("SECRET_KEY"):
```